crds/AzureBackendPool.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: azurebackendpools.appgw.ingress.azure.io
+spec:
+  group: appgw.ingress.azure.io
```

Comment:
Just to bring to notice, That we cannot use `azureingressprohibitedtargets.appgw.ingress.k8s.io` which was used for ProhibitedTarget CRD as sufficx `k8s.io` has been locked down.
https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190612-crd-group-protection.md